src/main/java/org/mule/service/oauth/internal/builder/AbstractOAuthDancerBuilder.java

```diff
@@ -110,11 +111,18 @@ public HttpResponse send(HttpRequest request, int responseTimeout, boolean follo
 
   @Override
   public OAuthDancerBuilder tokenUrl(String tokenUrl, TlsContextFactory tlsContextFactory) {
+    tokenUrl(tokenUrl, tlsContextFactory, null);
+    return this;
+  }
+
+  @Override
+  public OAuthDancerBuilder<D> tokenUrl(String tokenUrl, TlsContextFactory tlsContextFactory, ProxyConfig proxyConfig) {
```

Comment:
the implementation of `tokenUrl(String tokenUrl, ProxyConfig proxyConfig)` is missing